### PR TITLE
Cleanup in various files

### PR DESCRIPTION
**Pull request by jwillemsen**
Put all generated files in a subdirectory, currently failes due to issue #423

### CODE REVIEW

**Comment by jwillemsen** [comment]
Got something to work for the ReadCondition test

**Comment by mitza-oci** [comment]
Please split this into different PRs for the different independent changes.

The change in DDS_TAOv2.mwc seems to be already covered by PR #426.

I don't see the need for putting some "-*- MPC" at the top of files, people can configure their editors to recognize the file types based on names (*.mpc, *.mpb, etc.)

**Comment by jwillemsen** [comment]
Will zap the top MPC from the files. Let us wait with these changes after 3.10 has been released, I keep my changes on my git fork for the moment, need the QoS changes to continue my prototyping

File: MPC/config/dcps_qos_xml_handler.mpb [hunk 1]
```diff
@@ -1,5 +1,5 @@
-// -*- MPC -*-
-
-project : dcps_qos_xml_handler_base, dynamicany, typecodefactory {
-  requires += dds4ccm_opendds
+project : dynamicany, typecodefactory, dcps {
```
Comment (mitza-oci) [inline MPC/config/dcps_qos_xml_handler.mpb]: Are TAO's dynamic any and typecode factory libraries actually required?  They aren't listed in qos_xml_handler.mpc.

Comment (jwillemsen) [inline MPC/config/dcps_qos_xml_handler.mpb]: Correct, changed

**Comment by jwillemsen** [comment]
@mitza-oci travis/appveyor are green

**Comment by jwillemsen** [comment]
Looks to me that the single failing test is not related to my changes

**Comment by jwillemsen** [comment]
@mitza-oci Any remarks related to this PR?

**Comment by mitza-oci** [comment]
I'm looking at the MPC change now.  I have something simpler that works on gnuace but not on vc14.  This appears to be a bug in the MPC core for vc14.

**Comment by jwillemsen** [comment]
@mitza-oci Thanks!

**Comment by jwillemsen** [comment]
Merged master again into this PR, now has various cleanup

**Pull request merged**